src/executables/uber-time.js

```diff
@@ -6,14 +6,15 @@ import CommandExecutionService from '../services/CommandExecutionService';
 
 let service = new CommandExecutionService();
 
-program
-  .arguments('<address>')
-  .action(address => {
-    try {
-        service.executeTimeEstimates(address)
-               .then(table => console.log(table));
-    } catch (Error) {
-      console.error('Could not get time estimates');
+program.parse(process.argv);
+
+var address = program.args.toString();
```

Comment:
How do you feel about
```javascript
const address = program.args[0].trim();
```
I prefer using the actual index, in case arguments are ever added in the future. And the `trim()` should take care of the `uber time '    '` case.